packages/scandipwa/src/route/CartPage/CartPage.container.js

```diff
@@ -128,7 +128,7 @@ export class CartPageContainer extends PureComponent {
         }
 
         if (items_qty !== prevItemsQty) {
-            const title = `${ items_qty || '0' } Items`;
```

Comment:
Prefer using default argument value in destructuring declaration:
`const { a: myVar = 0 } = x`

Reply:
@yeegor please explain why requested changes preview include the outdated code, or what does current PR has to do with the default value since it is already supplemented? (`131: const title = __('%s Item(s)', items_qty || 0);`)